tests/examples/custom/custom.tf

```diff
@@ -13,12 +18,13 @@ provider "libvirt" {
 
 resource "libvirt_volume" "myvdisk" {
   name = "qatrfm-vdisk-${var.basename}-${count.index}.qcow2"
-  count = 2
   pool = "default"
-  source = "${var.image}"
+  count = 2
+  source = "${count.index == 0 ? var.image1 : var.image2}"
```

Comment:
Don't understand how this works, as we check for `image` as mendatory tf_var

Reply:
We only check for image as mandatory if no custom-files are provided. `image` is mandatory only for the default.tf
If you provide your own .tf file, it doesn't check for that. 

In this line I am making use of a nice feature of Terraform syntax. For the first domain, count.index=0,  and then source = var.image1... The second domain, count.index=1, and the source = var.image2 

It's a smart way to avoid creating 2 disk resources and saving a lot of lines and problems.

Reply:
nice